content/posts/js/introduction-gulp/index.md

```diff
@@ -185,45 +186,46 @@ gulp.task("default", ["styles"], function() {
 // Comme grunt, `gulp` sans argument lancera la tâche `default`.
 ```
 
-Bien entendu, vous avez déjà compris que si vous voulez remplacer Myth par Sass,
+Bien entendu, vous avez déjà compris que si vous voulez remplacer cssnext par Sass,
 c'est l'histoire de 4 secondes.
 
-## _Putain de code !_ approuve Gulp.
-
-Ici on est carrément fan de Gulp. Il faut bien avouer que
+Chez _Putain de code !_ on a aimé Gulp. Il faut bien avouer que
 [ça va vite](https://twitter.com/putaindecode/status/460868992396460032)
-(encore plus appréciable lorsque l'on n'a pas de SSD) et que c'est plaisant à écrire.
+(encore plus appréciable lorsque l'on n'a pas de SSD) et que c'est plaisant à
+écrire comparé à Grunt.
 Pas de configurations pas spécialement verbeuse et trop espacée.
 Avec Gulp on se sent plus libre, moins contraint.
-Du coup, **on a carrément fait notre site avec Gulp**
+Du coup, on avait carrément refait notre site avec Gulp
 (puis au passage un petit refresh graphique tant qu'à faire).
 
-Pour aller plus loin, vous n'avez qu'à ouvrir notre [Gulpfile](https://github.com/putaindecode/putaindecode.fr/blob/master/gulpfile.js)
-et regarder nos [tasks](https://github.com/putaindecode/putaindecode.fr/tree/master/tasks)
-qui vont de [la plus simple](https://github.com/putaindecode/putaindecode.fr/blob/master/tasks/clean.js)
-à [la plus compliqué](https://github.com/putaindecode/putaindecode.fr/blob/master/tasks/contributors.js).
+**Mise à jour: depuis nous avons simplifié encore simplifié notre process et nous
```

Comment:
Corrigé, merci.
https://github.com/putaindecode/putaindecode.io/commit/c875281140f3c6aba0ebfe438ec08687f6579d6b

A noté que tu aurais pu corriger via une PR :)